### PR TITLE
Add HTTP/3 service

### PR DESCRIPTION
**Pull request by jorti**
Expand the https service definition to include HTTP/3 (443/udp)

### CODE REVIEW

**Comment by erig0** [comment]
I'd rather see this as a new service called `http3`. There are many people still running http-1.1 servers so opening UDP does not make sense.

**Comment by jorti** [comment]
> I'd rather see this as a new service called `http3`.

Makes sense. I've updated the PR to create a new http3 service.

**Review by erig0 — APPROVED** [review]
lgtm. Thanks!
Waiting on CI before hitting merge.

**Review by erig0 — APPROVED** [review]

**Pull request merged**
